docker-compose.yml

```diff
@@ -74,7 +74,8 @@ services:
   aerie_sequencing:
     build:
       context: ./sequencing-server
-      dockerfile: Dockerfile.dev
+      dockerfile: Dockerfile
+      #dockerfile: Dockerfile.dev -- Uncomment for development
```

Comment:
Minor, nonblocking: If you use another `#` instead of a `--` here and L97, `Uncomment for development` will automatically become commented out when you uncomment the line.

Reply:
Good idea. I just made the change.